src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadYamlStep.java

```diff
@@ -60,6 +60,13 @@
  */
 public class ReadYamlStep extends AbstractFileOrTextStep {
 
+	public static final String DEFAULT_CODE_POINT_LIMIT_PROPERTY = ReadYamlStep.class.getName() + ".DEFAULT_CODE_POINT_LIMIT";
+
+	@SuppressFBWarnings(value={"MS_SHOULD_BE_FINAL"}, justification="Non final so that an admin can adjust the value through the groovy script console without restarting the instance.")
+	private static /*almost final*/ int DEFAULT_CODE_POINT_LIMIT = setDefaultCodePointLimit(Integer.getInteger(DEFAULT_CODE_POINT_LIMIT_PROPERTY, -1));
```

Comment:
This is very redundant. No need for a setter/getter for a pseudo constant.
```suggestion
	private static /*almost final*/ int DEFAULT_CODE_POINT_LIMIT = Integer.getInteger(DEFAULT_CODE_POINT_LIMIT_PROPERTY, -1);
```